pkg/drivemgr/loopbackmgr/Dockerfile.build

```diff
@@ -1,7 +1,7 @@
-FROM    ubuntu:21.04
+FROM    ubuntu:22.10
 
-RUN     sed -i -re 's/archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list \
-&&      sed -i -re 's/deb|deb-src/& \[trusted=yes\]/' /etc/apt/sources.list
+# RUN     sed -i -re 's/archive.ubuntu.com|security.ubuntu.com/old-releases.ubuntu.com/g' /etc/apt/sources.list \
+# &&      sed -i -re 's/deb|deb-src/& \[trusted=yes\]/' /etc/apt/sources.list
```

Comment:
why comment out these 2 lines?

Reply:
@libzhang because Ubuntu image was upgraded, and it's not an EOL release, so we don't use old-release.ubuntu .com as apt source.

if we keep it, "apt update" will fail.